src/main/java/ch/njol/skript/effects/EffEquip.java

```diff
@@ -93,19 +81,22 @@ public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelaye
 		return true;
 	}
 
-	private static final boolean SUPPORTS_STEERABLE = Skript.classExists("org.bukkit.entity.Steerable");
-
 	private static ItemType CHESTPLATE;
 	private static ItemType LEGGINGS;
```

Comment:
A note mainly for the members, but is there any use in ItemType over Tag references? Aliases wouldn't matter as far as I see and Itemtype already does more each time than doing `tag.isTagged(Item.getType())`

Reply:
compatibility
![image](https://github.com/user-attachments/assets/1e57f17c-4520-4e4b-8a50-0beb0b91ab83)

Reply:
Ahh alright I see now okay